scripts/base/frameworks/cluster/agent/boot.zeek

```diff
@@ -0,0 +1,35 @@
+@load base/frameworks/cluster/agent/config
+
+# The agent needs the supervisor to listen for node management requests.  We
+# need to tell it to do so, and we need to do so here, in the agent
+# bootstrapping code, so the redef applies prior to the fork of the agent
+# process itself.
+redef SupervisorControl::enable_listen = T;
+
+event zeek_init()
+	{
+	if ( ! Supervisor::is_supervisor() || ! ClusterAgent::enable )
+		return;
+
+	local epi = ClusterAgent::endpoint_info();
+	local sn = Supervisor::NodeConfig($name=epi$id,
+		$scripts=vector("base/frameworks/cluster/agent/main.zeek"));
```

Comment:
Suggest to name `main.zeek` differently as the normal convention is for `__load__.zeek` to pull in `main.zeek`, which isn't the case here. Maybe even use another directory level to separate things out into what's always loaded vs what not.

Reply:
Hah. Until two days ago it was "runtime.zeek". :-) Do you prefer that? I figured there should be a `main.zeek` somewhere, since it's what people are trained to look for the essence of the functionality, and that's closest.